lib/grizzly/zipgateway/config.ex

```diff
@@ -194,4 +210,17 @@ defmodule Grizzly.ZIPGateway.Config do
       config_string
     end
   end
+
+  defp rf_region(:eu), do: "0x00"
```

Comment:
```suggestion
  defp rf_region(:eu), do: 0x0
```

I just checked the zipgateway source code and it uses `strtoul` to parse the region number. That means you can have regular integers here rather than string hex numbers.